src/utils/command.ts

```diff
@@ -60,7 +69,13 @@ export function parseArgs(
       case 'user':
       case 'role':
       case 'channel':
-        parseMention(args, entry, messageArgsNullableCopy)
+        await parseMention(
+          args,
+          entry,
+          messageArgsNullableCopy,
+          message.client,
```

Comment:
maybe use one message arg right below and get client in `parseMention`

Reply:
Like don't pass message.client as parameter.
But replace `client` in the code with `message.client`?

Reply:
yes